viral/ensemble_reactivation.py

```
@@ -783,7 +797,13 @@ def main(mouse: str, date: str, plot: bool = True) -> None:
             flu=place_cells, wheel_freeze=session.wheel_freeze
         )
 
-        trials = [trial for trial in session.trials if trial_is_imaged(trial)]
+        trials = [
+            trial
+            for trial in session.trials
+            if trial_is_imaged(trial)
+            and (rewarded is None)
```

Ruff (0.14.0)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)